src/models/SpectralDefinition.ts

```diff
@@ -60,15 +60,15 @@ export enum SpectralSystem {
     BARY = "BARYCENT",
     TOPO = "TOPOCENT"
 }
-export const IsSpectralSystemValid = (system: string): boolean => {
+export const IsSpectralSystemSupported = (system: string): boolean => {
     return system && (<any> Object).values(SpectralSystem).includes(system);
 };
 
 export const SPECTRAL_TYPE_STRING = new Map<SpectralType, string>([
-    [SpectralType.VRAD, "Radio velocity"],
-    [SpectralType.VOPT, "Optical velocity"],
+    [SpectralType.VRAD, "Velocity"],
```

Comment:
shouldn't we stick with the WCS Paper III naming conventions for these?

Reply:
I am trying to unify the names according to CHANNEL_TYPES(following FITS standard V4.0), since the "Native WCS" label in dropdown will compare the strings of names

Reply:
Yes, that's what I'm suggesting as well:
![image](https://user-images.githubusercontent.com/592504/75865581-7132e200-5e0c-11ea-9436-41bab25159dc.png)

Reply:
hmm I see, will change it